specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/getComputeOneSkuUsages.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "scope": "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus",
+    "resourceName": "standardNDSFamily",
```

Comment:
>standardNDSFamily [](http://example.com/codeflow?start=21&length=17)

The response should have a top level 'name' property mathcing the resource name from the request.

It's okay to keep the name property inside "properties" since it looks like you want to provide a localized value. 

Please add the top level name property for all examples.

Reply:
I will add the top level name. It looks like duplicate property.